QUIZ/quiz1-program8.ipynb

```diff
@@ -0,0 +1,99 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "id": "d3153549",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "eab9f4d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "header=[\"name\",\"contact\"]\n",
+    "\n",
+    "data = [[\"om\", \"9316203288\"],\n",
+    "        [\"sai\",\"8767564567\"],\n",
+    "        [\"ram\", \"8967567890\"],\n",
+    "        [\"radhe\", \"9067567890\"],\n",
+    "        [\"krishna\", \"9167567890\"]]\n",
+    "filename = \"Contacts.csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "cf6e292c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('filename','w',newline='') as f:\n",
+    "    k=csv.writer(f,delimiter='|')\n",
+    "    k.writerow(header)\n",
+    "    k.writerows(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "id": "96cc3577",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['name', 'contact']\n",
+      "['om', '9316203288']\n",
+      "['sai', '8767564567']\n",
+      "['ram', '8967567890']\n",
+      "['radhe', '9067567890']\n",
+      "['krishna', '9167567890']\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open('filename','r') as r:\n",
+    "    s=csv.reader(r,delimiter='|')\n",
```

Comment:
what is delimiter-=' | '?

Reply:
specify character used to separate data or field.